### PR TITLE
[MAJOR] Updated a lot of npm packages

### PR DESCRIPTION
**Pull request by goldyydev**
This PR
 - Fixes **all** of the ESLint styling problems we had <sub><i>(please stop using GitHub desktop)<i/><sub/>
 - Bumps all of the packages that needed upgrades. that includes a few major upgrades
 - [ ]  **Needs to be tested by more people than me to make sure this has no compatibility issues**

### CODE REVIEW

File: kibbeh/package.json [hunk 1]
```diff
@@ -17,53 +17,53 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@date-io/date-fns": "^1.3.13",
+    "@date-io/date-fns": "^2.10.8",
```
Comment (benawad) [inline kibbeh/package.json]: this needs to stay at v1 its specified in readme for lib we are using

File: kibbeh/package.json [hunk 2]
```diff
@@ -17,53 +17,53 @@
...
-    "date-fns": "^2.19.0",
+    "date-fns": "^2.20.1",
     "emoji-regex": "^9.2.2",
     "formik": "^2.2.6",
     "grapheme-splitter": "^1.0.4",
     "hark": "^1.2.3",
-    "i18next": "^19.9.2",
-    "i18next-browser-languagedetector": "^6.0.1",
-    "i18next-http-backend": "^1.1.1",
+    "i18next": "^20.2.1",
```
Comment (benawad) [inline kibbeh/package.json:31]: if you are going to upgrade major version, check to make sure there are no breaking changes

**Review by benawad — CHANGES_REQUESTED** [review]

File: kibbeh/package.json (hunk 2 above)
Comment (goldyydev) [reply kibbeh/package.json:31]: I already checked and it appears as if translations are working

Comment (goldyydev) [reply kibbeh/package.json:31]: image

Comment (benawad) [reply kibbeh/package.json:31]: no I mean read the changelog for i18next

Comment (goldyydev) [reply kibbeh/package.json:31]: This shouldn't break anything

File: kibbeh/package.json (hunk 1 above)
Comment (goldyydev) [reply kibbeh/package.json]: Fixed

**Comment by goldyydev** [comment]
@benawad I resolved all conflicts and fixed all issues, this should be good to go.

**Pull request merged**